### ISSUE TITLE
whats app share is not working in firefox browser

### ISSUE THREAD

**Issue by shivasai09** [nygardk/react-share#196]
```
<WhatsappShareButton
   url={`https://mywebsite.com`}
   quote="hip hip hurray"                 
    >
```

the above is the code we are using to share some text to the whatsapp from my website which is not working in the fire fox

![image](https://user-images.githubusercontent.com/33802398/49509798-65c0d600-f8ac-11e8-9ac8-c92a5ab46b3a.png)

**Comment by aautio** [comment]
This is a duplicate of #179 and #157. It might get fixed when we land #194

**Comment by joyblanks** [comment]
When are you planning to release this?
